app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ActiveRecord::Base
+  belongs_to :commentable, polymorphic: true
+  belongs_to :user
+
+  validates :content, :user_id, :commentable_type, :commentable_id, presence: true
```

Comment:
Prefer validating the object over the id - checks that the object exists (i.e. that the id is valid.)  Also you can validate the commentable in one

so `validates :content, :user, :commentable, presence: true` should work.